lib/browser/io.ts

```diff
@@ -23,6 +23,23 @@ type BodyData = string | object | ArrayBuffer | ArrayBufferView | Blob | File;
 
 export class InputStream implements InputStreamBase {
     constructor(public data: BodyData) {
-        
+
+    }
+}
+
+/**
+ * Standard Berkeley socket style options.
+ *
+ * Provided for compatibility with nodejs, but this version is largely unused.
+*/
+export class SocketOptions {
+    constructor(
+        public type: SocketType = SocketType.STREAM,
+        public domain: SocketDomain = SocketDomain.IPV6,
```

Comment:
Shouldn't this be IPV4?

Reply:
It's IPV6 in Python, so I matched that. I could go either way on this though.